.travis.yml

```diff
@@ -12,11 +12,12 @@ gemfile:
   - gemfiles/rails_4.gemfile
   - gemfiles/rails_5.gemfile
 script:
-- bundle exec rspec
-- bundle exec codeclimate-test-reporter
+  - bundle exec rspec
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: 2.4.0
       gemfile: gemfiles/rails_3.gemfile
-
+addons:
+  code_climate:
+    repo_token: b65d6447fdd48601786f688ef382bd56fd597b906943ff5c27d3d0fbf1bfc966
```

Comment:
good to know thanks. i think that bash script has more to do with scanning the reports and combining them together (the only we tested that actually had multiple language support -> ruby & JS).